jmxfetch.py

```diff
@@ -37,93 +42,100 @@
 ]
 JMX_COLLECT_COMMAND = 'collect'
 JMX_LIST_COMMANDS = {
-        'list_everything': 'List every attributes available that has a type supported by JMXFetch',
-        'list_collected_attributes': 'List attributes that will actually be collected by your current instances configuration',
-        'list_matching_attributes': 'List attributes that match at least one of your instances configuration',
-        'list_not_matching_attributes': "List attributes that don't match any of your instances configuration",
-        'list_limited_attributes': "List attributes that do match one of your instances configuration but that are not being collected because it would exceed the number of metrics that can be collected",
-        JMX_COLLECT_COMMAND: "Start the collection of metrics based on your current configuration and display them in the console"
-        }
+    'list_everything': 'List every attributes available that has a type supported by JMXFetch',
+    'list_collected_attributes': 'List attributes that will actually be collected by your current instances configuration',
+    'list_matching_attributes': 'List attributes that match at least one of your instances configuration',
+    'list_not_matching_attributes': "List attributes that don't match any of your instances configuration",
+    'list_limited_attributes': "List attributes that do match one of your instances configuration but that are not being collected because it would exceed the number of metrics that can be collected",
+    JMX_COLLECT_COMMAND: "Start the collection of metrics based on your current configuration and display them in the console"}
 
 PYTHON_JMX_STATUS_FILE = 'jmx_status_python.yaml'
 
 LINK_TO_DOC = "See http://docs.datadoghq.com/integrations/java/ for more information"
 
-class InvalidJMXConfiguration(Exception): pass
 
-class JMXFetch(object):
+class InvalidJMXConfiguration(Exception):
+    pass
+
+
+class JMXFetch(Daemon):
```

Comment:
Should this be a daemon ? It's probably not needed, supervisor will "daemonize" the process.